### PR TITLE
Allow skip Journal for Apache Pulsar

### PR DESCRIPTION
**Pull request by codelipenghui**
- Allow skip Journal by using -e skip_journal=true
- Fix Pulsar dashboard

### CODE REVIEW

File: driver-pulsar/deploy/ssd/templates/bookkeeper.conf [hunk 1]
```diff
@@ -292,15 +296,15 @@ auditorPeriodicBookieCheckInterval=86400
 # number of threads that should handle write requests. if zero, the writes would
 # be handled by netty threads directly.
 numAddWorkerThreads=8
```
Comment (merlimat) [inline driver-pulsar/deploy/ssd/templates/bookkeeper.conf:298]: By default the `numAddWorkerThreads=8` should actually be 0 so we directly write from the IO threads

File: driver-pulsar/pulsar.yaml [hunk 2]
```diff
@@ -24,7 +24,7 @@ driverClass: io.openmessaging.benchmark.driver.pulsar.PulsarBenchmarkDriver
 client:
   serviceUrl: pulsar://localhost:6650
   httpUrl: http://localhost:8080
-  ioThreads: 8
+  ioThreads: 4
```
Comment (merlimat) [inline driver-pulsar/pulsar.yaml:27]: Why reducing IO threads?

**Comment by codelipenghui** [comment]
@merlimat I have reverted the unrelated change, I think it is from my previous test

**Review by merlimat — APPROVED** [review]

File: driver-pulsar/deploy/ssd/templates/pulsar_env.sh [hunk 3]
```diff
@@ -45,7 +45,7 @@
 PULSAR_MEM=" -Xms{{ max_heap_memory }} -Xmx{{ max_heap_memory }} -XX:MaxDirectMemorySize={{ max_direct_memory }}"
 
 # Garbage collection options
-PULSAR_GC=" -XX:+UseG1GC -XX:MaxGCPauseMillis=10 -XX:+ParallelRefProcEnabled -XX:+UnlockExperimentalVMOptions -XX:+AggressiveOpts -XX:+DoEscapeAnalysis -XX:ParallelGCThreads=32 -XX:ConcGCThreads=32 -XX:G1NewSizePercent=50 -XX:+DisableExplicitGC -XX:-ResizePLAB"
+PULSAR_GC=" -XX:+UnlockExperimentalVMOptions -XX:+UseZGC -XX:+ParallelRefProcEnabled -XX:+AggressiveOpts -XX:+DoEscapeAnalysis -XX:ParallelGCThreads=12 -XX:ConcGCThreads=12 -XX:+DisableExplicitGC -XX:-ResizePLAB"
```
Comment (lhotari) [inline driver-pulsar/deploy/ssd/templates/pulsar_env.sh:48]: It seems that there's a switch to use ZGC. @codelipenghui  What is the impact of the change alone from G1GC -> ZGC?

Btw. Are you aware of https://bugs.openjdk.java.net/browse/JDK-8257145 ? It's better to remove `-XX:-ResizePLAB` option on JDK 11+.

**Pull request merged**